src/utils/serializers/bool.ts

```diff
@@ -2,6 +2,7 @@
  * Converts a Uint8Array to boolean.
  *
  * @param arr - the Uint8Array to convert
+ * @return {boolean} the converted boolean
```

Comment:
This is type script TSDoc so no JSDoc type annotation required, and stop forgetting the new line between different tags types :sweat_smile: